test/FormElements.test.ts

```diff
@@ -1,4 +1,6 @@
 import { expect, fixture, html } from '@open-wc/testing';
+import '@material/web/radio/radio.js'
```

Comment:
drive-by comment, I wouldn't recommend importing a final implementation of a component, there's a lot going on in MWC's radio that would make this difficult to reliably test.

Instead, I'd try to see what `<md-radio>` is doing that isn't working and create a minimal reproduction class like `TestInput`.